plutus-core/untyped-plutus-core/test/Analysis/Spec/Lib.hs

```diff
@@ -0,0 +1,76 @@
+{-# LANGUAGE OverloadedStrings #-}
+
+module Analysis.Spec.Lib where
```

Comment:
I don't think there's any existing module named "*.Spec.Lib". For consistency, just call it `Analysis.Lib`?